ocaml/lib/pickles_bindings.mli

```diff
@@ -30,6 +30,7 @@ type pickles_rule_js =
           ; shouldVerify : Boolean.var array Js.prop >
           Js.t )
       Js.prop
+  ; featureFlags : bool Pickles_types.Plonk_types.Features.t Js.prop
```

Comment:
I would be in favor of documenting a bit the parameters as we are in an interface file :)

Reply:
I'm not sure if this logic applies here, because this interface is only used from JS, not from OCaml, so the comments won't enhance any editor experience. Probably well documented interfaces on the JS side are a better investment

Reply:
Regarding the interface, it will still be used by the o1js maintainer.

Reply:
addressed here instead because this is where the interface is exposed: https://github.com/o1-labs/o1js/pull/1176/commits/68b9ba89d4daeeb59f5e93e65c557be3d6d96328